AzurePipelineDecorator/cli:simple-example/snyk-pipeline-deco-cli.yml

```
@@ -0,0 +1,12 @@
+steps:
+- task: CmdLine@2
+  displayName: 'Decorator-injected Task to test Snyk using CLI'
+  inputs:
+    script: |
+      sudo npm install -g snyk
+      sudo npm install snyk-to-html -g
```

```suggestion
      sudo npm install -g snyk-to-html
```
to keep it the same pattern as the above line